compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -754,6 +754,36 @@ TR::Register *OMR::X86::TreeEvaluator::sloadEvaluator(TR::Node *node, TR::CodeGe
    return reg;
    }
 
+TR::Register *iReadBarrierEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return OMR::X86::TreeEvaluator::iloadEvaluator(node,cg);
```

Comment:
It may be better to use `TR::TreeEvaluator::iloadEvaluator` so that it can reach the `iloadEvaluator` overridden by downstream projects.

Reply:
changed according to the comments.